pkg/apis/flowcontrol/v1beta3/defaults.go

```diff
@@ -61,8 +83,7 @@ func SetDefaults_LimitedPriorityLevelConfiguration(lplc *v1beta3.LimitedPriority
 	}
 }
 
-// SetDefaults_FlowSchema sets default values for flow schema
-func SetDefaults_QueuingConfiguration(cfg *v1beta3.QueuingConfiguration) {
```

Comment:
keep this function as-is, SetObjectDefaults_PriorityLevelConfiguration will still call it